keps/sig-apps/2214-indexed-job/README.md

```diff
@@ -259,6 +353,15 @@ The Job controller doesn't add the environment variable if there is a name
 conflict with an existing environment variable. Users can specify other
 environment variables for the same annotation.
 
+<<[UNRESOLVED this deviates from the rest of the controllers ]>>
+The Pod name takes the form `$(job-name)-$(index)-$(random-string)`,
```

Comment:
Why do we need to change pod name itself (given we're going to set hostname)?

[Also - it contains random suffix, so I don't see any place where it helps.
If people really want to fetch a pod for a given index, we should be doing this by label selector.]

Reply:
I think this is useful, especially when looking at logs, and has no downsides. So I am in favor of this change.

Reply:
Yes, this is just for the purpose of debugging. Added the notes.

Reply:
What is the plan with this unresolved item? Do we want to leave it as is or remove it? I'd prefer not to merge a KEP with an unresolved element.

Reply:
I personally think we can just resolve it.

I'm not a huge fan of it, but I don't think there is any strong enough reason for not doing this.

Reply:
The plan was to wait for feedback before merging. Since there doesn't seem to be opinions against it, I removed the unresolved tags.